### PR TITLE
Bump dependencies and prepare v3.13.0 release

### PR DESCRIPTION
**Pull request by nikku**
Closes https://github.com/camunda/camunda-bpmn-js/issues/312

### CODE REVIEW

**Comment by nikku** [comment]
Found during integration testing: https://github.com/bpmn-io/bpmn-js-properties-panel/issues/1018.

**Comment by nikku** [comment]
Found during integration testing: https://github.com/bpmn-io/bpmn-js-element-templates/blob/main/CHANGELOG.md#1131.

**Comment by nikku** [comment]
Found during integration testing: https://github.com/bpmn-io/bpmn-js-properties-panel/issues/1021.

**Comment by nikku** [comment]
Found another one, which was due to the REST connector template `URL#pattern` to be too rigid: 

capture i5WfH1_optimized

Fixed via https://github.com/camunda/camunda-bpmn-js/pull/340/commits/ba7cc36cbe57623322965cda4cfcd176976b8330.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,24 +8,44 @@ ___Note:__ Yet to be released changes appear here._
...
+* `FIX`: do not set label on planes ([bpmn-io/bpmn-js#2033](https://github.com/bpmn-io/bpmn-js/issues/2033))
 
 ### Key Changes in Properties Panel
 
+* `FEAT`: improve Camunda 7 variable events tooltip ([#1016](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/1016))
+* `FEAT`: simplify FEEL editor external error ([bpmn-io/properties-panel#97](https://github.com/camunda/linting/pull/97))
+* `FEAT`: remove unnecessary resizer ([bpmn-io/bpmn-js-properties-panel@`b2f6752`](https://github.com/bpmn-io/properties-panel/commit/b2f6752de3827384452d4b4c0b27bd269b7b5ad4))
 * `FEAT`: display element template icon found in XML (https://github.com/bpmn-io/bpmn-js-properties-panel/issues/1011)
-chore: align zeebe input propagation label and tooltip (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/5d8bd6846efcbbc7c67322df5a6c6fe28d63fb9b)
+* `FEAT`: align zeebe input propagation label and tooltip (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/5d8bd6846efcbbc7c67322df5a6c6fe28d63fb9b)
```
Comment (marstamm) [inline CHANGELOG.md:30]:
```suggestion
* `FEAT`: align zeebe input propagation label and tooltip ([bpmn-io/bpmn-js-properties-panel@`5d8bd68`](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/5d8bd6846efcbbc7c67322df5a6c6fe28d63fb9b))
```

**Review by marstamm — APPROVED** [review]
🚀 Great work!

**Comment by nikku** [comment]
Found a few more changelog broken links. Thanks for the review.

**Pull request merged**